starknet_py/net/l1/messages.py

```diff
@@ -80,20 +84,20 @@ def from_content(cls, msg_content: MessageToEthContent) -> "MessageToEth":
         return cls.from_hash(msg_content.hash)
 
     @classmethod
-    def from_tx_receipt(cls, tx_receipt) -> List["MessageToEth"]:
+    def from_tx_receipt(cls, tx_receipt: TransactionReceipt) -> List["MessageToEth"]:
         """
         :param tx_receipt: A JSON from starknet.py's Client `get_transaction_receipt`
         :return: A list of Starknet to Ethereum messages
         """
         return [
             MessageToEth.from_content(
                 MessageToEthContent(
-                    starknet_sender=message["from_address"],
-                    eth_recipient=int(message["to_address"], 16),
-                    payload=[int(felt_str) for felt_str in message["payload"]],
+                    starknet_sender=message.from_address,
+                    eth_recipient=int(message.to_address, 16),
+                    payload=[int(felt_str) for felt_str in message.payload],
                 )
             )
-            for message in tx_receipt["l2_to_l1_messages"]
+            for message in tx_receipt.l2_to_l1_messages
         ]
 
     @classmethod
```

Comment:
Lines 133 and 249 have incorrect typing: both `l1_to_l2_message` and `l2_to_l1_message` return int according to typing that is then cast `int_from_hexbytes` that doesn't make sense. Should we remove casting or change typing here? 
cc @Arcticae

Reply:
we should probably change typing. keeping the interface intact seems the way to go